jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/inputGroup/InputGroupButtonAddonsTests.java

```diff
@@ -0,0 +1,57 @@
+package io.github.epam.bootstrap.tests.composite.section.inputGroup;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.elements.init.UIFactory.$;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.*;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+
+public class InputGroupButtonAddonsTests extends TestsInit {
+    private static final String JS_SCROLL_TO_ELEMENT = "scrollIntoView(true);";
+    private static final String HEADER_RIGHT_PART = "//h4[.='Input group - Button addons']";
+    private String text = "XA-XA",
+       placeholder_text = "Recipient's username";
```

Comment:
it's not a common case to use comma, possibly it's better to follow the general style and define as usual we define field

Reply:
placeholder_text should be lower camel case